code/modules/clothing/spacesuits/rig/modules/ds13/kinesis/extension.dm

```diff
@@ -13,8 +13,8 @@
 	var/cached_plane
 
 	//The graphical filter we assigned to the held item
-	var/dm_filter/ripple
```

Comment:
no don't do this, dm_filter is a valid native type, just hidden
there should be a file somewhere which declares it to prevent compiler errors, i don't know why dreamchecker isn't seeing that